src/arithmetic.jl

```diff
@@ -25,6 +30,14 @@ import Base: +, -, *
 *(x::Number, M::IntervalMatrix) = Interval(x) * M
 *(M::IntervalMatrix, x::Number) = Interval(x) * M
 
+# TODO: continue here
```

Comment:
I think you can remove this line - it will not be clear in a few months (it is not even clear to me now). Rather open an issue.
```suggestion
```

Reply:
Thanks; i just forgot to remove this line after i "continued here" :)